llvm-external-projects/iree-dialects/lib/Dialect/IREEPyDM/Transforms/ToIREE/LoweringPatterns.cpp

```diff
@@ -140,6 +242,95 @@ class BoxOpConversion : public OpConversionPattern<pydm_d::BoxOp> {
   }
 };
 
+class BranchConversion : public OpConversionPattern<std_d::BranchOp> {
+  using OpConversionPattern::OpConversionPattern;
+  LogicalResult matchAndRewrite(
+      std_d::BranchOp srcOp, OpAdaptor adaptor,
+      ConversionPatternRewriter &rewriter) const override {
+    rewriter.replaceOpWithNewOp<std_d::BranchOp>(srcOp, srcOp.dest(),
+                                                 adaptor.destOperands());
+    return success();
+  }
+};
+
+class CallOpConversion : public OpConversionPattern<pydm_d::CallOp> {
+  using OpConversionPattern::OpConversionPattern;
+
+  LogicalResult matchAndRewrite(
+      pydm_d::CallOp srcOp, ArrayRef<Value> operands,
+      ConversionPatternRewriter &rewriter) const override {
+    pydm_d::CallOp::Adaptor adaptor(operands);
+    SmallVector<Type> resultTypes;
+    if (failed(getTypeConverter()->convertTypes(srcOp.getResultTypes(),
+                                                resultTypes))) {
+      return rewriter.notifyMatchFailure(srcOp,
+                                         "result types could not be converted");
+    }
+    rewriter.replaceOpWithNewOp<std_d::CallOp>(srcOp, srcOp.callee(),
+                                               resultTypes, adaptor.operands());
+    return success();
+  }
+};
+
+class CondBranchConversion : public OpConversionPattern<std_d::CondBranchOp> {
+  using OpConversionPattern::OpConversionPattern;
+  LogicalResult matchAndRewrite(
+      std_d::CondBranchOp srcOp, OpAdaptor adaptor,
+      ConversionPatternRewriter &rewriter) const override {
+    rewriter.replaceOpWithNewOp<std_d::CondBranchOp>(
+        srcOp, adaptor.condition(), srcOp.trueDest(),
+        adaptor.trueDestOperands(), srcOp.falseDest(),
+        adaptor.falseDestOperands());
+    return success();
+  }
+};
+
+class ConstantOpConversion : public OpConversionPattern<pydm_d::ConstantOp> {
+  using OpConversionPattern::OpConversionPattern;
+
+  LogicalResult matchAndRewrite(
+      pydm_d::ConstantOp srcOp, OpAdaptor adaptor,
```

Comment:
FYI these aliases feel like a readability negative (as well as making things trickier to large-scale change) - I'd prefer not having this style leak outside of these files if you find yourself convinced they are good and I'll just hold my nose here :)
(I can understand having the arith & co namespaced as those are likely to end up in their own namespaces and this will be easy to swap out with the appropriate one when done, but for permanent namespaces the extra 2-3 characters saved during initial typing don't feel worth the naming ambiguity/divergence from definition)

Reply:
Yeah, I don't disagree or agree yet. Let's not let this escape and I will take it as a feedback point that the namespaces need to be cleaned up -- would just like to do that once and not piecemeal. The high degree of overlap that these dialects have (naming wise) pushed me to a "qualify everything" approach, and then I ran into annoying nesting issues. Needs a rethink.